Extensions/Today/TodayUX.swift

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+struct TodayUX {
+
+    static let backgroundHightlightColor = UIColor(white: 216.0/255.0, alpha: 44.0/255.0)
+    static let linkTextSize: CGFloat = 9.0
+    static let labelTextSize: CGFloat = 12.0
+    static let imageButtonTextSize: CGFloat = 13.0
+    static let copyLinkImageWidth: CGFloat = 20
+    static let margin: CGFloat = 8
+    static let buttonsHorizontalMarginPercentage: CGFloat = 0.1
+    static let buttonStackViewSpacing: CGFloat = 20.0
+    static var labelColor: UIColor {
+        if #available(iOS 13, *) {
+            return UIColor(named: "widgetLabelColors") ?? UIColor(rgb: 0x242327)
+        } else {
+            return UIColor(rgb: 0x242327)
+        }
+    }
+    static var subtitleLabelColor: UIColor {
+        if #available(iOS 13, *) {
+            return UIColor(named: "subtitleLableColor") ?? UIColor(rgb: 0x38383C)
+        } else {
+            return UIColor(rgb: 0x38383C)
+        }
+    }
+}
+
+struct TodayStrings {
```

Comment:
We should move this from TodayStrings to Strings.swift file 
```extension Strings {
    public static let NewPrivateTabButtonLabel = NSLocalizedString("TodayWidget.NewPrivateTabButtonLabel", tableName: "Today", value: "Private Search", comment: "New Private Tab button label")
    public static let NewTabButtonLabel = NSLocalizedString("TodayWidget.NewTabButtonLabel", tableName: "Today", value: "New Search", comment: "New Tab button label")
    public static let GoToCopiedLinkLabel = NSLocalizedString("TodayWidget.GoToCopiedLinkLabel", tableName: "Today", value: "Go to copied link", comment: "Go to link on clipboard")
}
```

In the code you could use it as 
```
imageButton.label.text = Strings.NewTabButtonLabel
```

Reply:
yes, Planning to make a separate PR for the strings 👍🏻